### PR TITLE
Add error message and remove duplications.

### PR DESCRIPTION
**Pull request by sima-zhu**
We have duplications when read the toc map. Remove duplication. Also print error message.

image

### CODE REVIEW

**Comment by danieljurek** [comment]
What's causing these duplications?

**Comment by sima-zhu** [comment]
> What's causing these duplications?

What we do is only read the toc.yml generated from Docs site tool.
```
def grep_children_namespaces(autogenerated_toc_yml):
    return [
        top_level_namespace["name"] for top_level_namespace in autogenerated_toc_yml
    ] + ["*"]
```

We have a step for `Merge Un-Migratable Legacy Packages To Stable ToC` which read APIs from [here](https://github.com/Azure/azure-docs-sdk-java/blob/main/temporary-src-based-yml/migration.json)

File: scripts/python/docs-automation/sync_tocs.py [hunk 1]
```diff
@@ -202,15 +202,15 @@ def grep_children_namespaces(autogenerated_toc_yml):
...
+            "Execution requires that both the known namespaces and reference yml be defined. Error message: " + str(f)
         )
 
     present_in_target = grep_children_namespaces(target_autogenerated_toc)
 
     print(
         "Here are the visible namespaces in target autogenerated ToC. Constraining reference.yml."
     )
-    for ns in sorted(present_in_target):
+    for ns in sorted(list(dict.fromkeys(present_in_target))):
```
Comment (scbedd) [inline scripts/python/docs-automation/sync_tocs.py:213]: ❤️

**Review by scbedd — APPROVED** [review]

File: scripts/python/docs-automation/sync_tocs.py (hunk 1 above)
Comment (scbedd) [reply scripts/python/docs-automation/sync_tocs.py:213]: > > What's causing these duplications?
> 
> What we do is only read the toc.yml generated from Docs site tool.
> 
> ```
> def grep_children_namespaces(autogenerated_toc_yml):
>     return [
>         top_level_namespace["name"] for top_level_namespace in autogenerated_toc_yml
>     ] + ["*"]
> ```
> 
> We have a step for `Merge Un-Migratable Legacy Packages To Stable ToC` which read APIs from [migration.json](https://github.com/Azure/azure-docs-sdk-java/blob/main/temporary-src-based-yml/migration.json)
> 
> The duplication APIs from all from this migration.json file.

Have you cleaned any packages that are in `latest` now out of the `migration.json`? If there is duplication that indicates to me that we are _overwriting_ the `latest` bits with output from the migration.json file.

**Review by scbedd — CHANGES_REQUESTED** [review]
See my latest comment. The fact that you're seeing duplication strikes me as incorrect. I agree with the bugfix in principle still.

**Comment by sima-zhu** [comment]
@scbedd I will compare latest toc.yml with migration.json, and remove duplicate ones from migration.json. Close the PR and will open another one in docs repo.

**Pull request closed without merging**